app/assess/templates/macros/comments_box.html

```diff
@@ -15,6 +15,9 @@
       "classes": "govuk-label--s",
       isPageHeading: true
     },
+    "errorMessage": {
+      "text": "A comment must not be empty"
+  } if commentForm.comment.errors
```

Comment:
Could be too long or empty so this should handle both cases, can we pass through the errors we get?

Reply:
Could do a catch all wording like "A comment must be between 1-1000 characters"